src/main/java/io/fusionauth/load/FusionAuthWorkerFactory.java

```diff
@@ -74,5 +74,27 @@ public Worker createWorker() {
 
   @Override
   public void prepare(LoadDefinition loadDefinition) {
+    String directive = null;
```

Comment:
+1 

Like this a lot better since this is where we define and use `directive`, `apiKey`, `url` etc. 
Thanks!